src/utils.js

```diff
@@ -0,0 +1,15 @@
+findHighest = (a,b,c) => {
```

Comment:
We can export directory from here by saying

```
exports.findHighest = (a,b,c) => {
  ...
  ...
}
```

and then we **WONT** need the line
```
exports.findHighest = findHighest;
```

Reply:
have done that